File: Day5/Subtract the Product and Sum of Digits of an Integer/1281.cs
public class Solution {
    public int SubtractProductAndSum(int n) {
        var sum = 0;
        var product = 1;
        
        while(n != 0)
        {
            var digit = n % 10;
            
            sum += digit;
            product *= digit;
            
            n /= 10;
        }
        
        return product - sum;
    }
}

File: Day5/Subtract the Product and Sum of Digits of an Integer/1281.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        sum = 0
        product = 1
        
        while n != 0:
            digit = n % 10
            
            sum += digit
            product *= digit
            
            n //= 10
        
        return product - sum
        